disputer/disputer.js

```diff
@@ -224,6 +238,11 @@ class Disputer {
         txnConfig
       });
 
+      // Before submitting transaction, store liquidation timestamp before it is potentially deleted if this is the final reward to be withdrawn.
+      // We can be confident that `liquidationTime` property is available and accurate because the liquidation has not been deleted yet if we `withdrawLiquidation()`
+      // is callable.
+      const requestTimestamp = liquidation.liquidationTime;
```

Comment:
Note: fetching `liquidtion.liquidationTime` is a bit hacky here and we can only assume it will work because we've already checked that `withdrawLiquidation()` will succeed. This implies that the liquidation struct has not been deleted on-chain, so the `liquidationTime` property is still intact.

Reply:
This is subtle -- very nice. Seems like a completely reasonable assumption.